userspace/libc/src/printf.c

```diff
@@ -495,7 +495,7 @@ extern int printf(const char *format, ...)
       if(*format == '.') {
         ++format; 
         char num[4];
-        c = 0;
+        size_t c = 0;
```

Comment:
_(note: unrelated build fix after #138, which broke this variable reuse due to scoping change)_